bridge.go

```diff
@@ -341,7 +342,12 @@ func (h *Helper) HTTPCallRawWithOptsWithContext(ctx context.Context, method, url
 	if err != nil {
 		return nil, err
 	}
-	req.Header.Add("Content-Type", "application/json")
+
+	if (opts.ContentType == "") {
```

Comment:
Trivial change, but personally prefer `len(opts.ContentType) > 0` in Go, as it looks a bit cleaner rather than literal string matching.